components/tools/OmeroWeb/test/integration/utils.py

```diff
@@ -0,0 +1,31 @@
+
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright (C) 2020 University of Dundee & Open Microscopy Environment.
+# All rights reserved.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+def rgba_to_int(red, green, blue, alpha=255):
```

Comment:
From recent work IDR companion files, I discovered that with Python 3 the whole block below is now part of the standard library - see [int.from_bytes](https://docs.python.org/3/library/stdtypes.html?highlight=bitwise#int.from_bytes). This can be replaced by a one-liner

```
     return int.from_bytes((red, green, blue, alpha), byteorder='big', signed=True)
```

Reply:
that is useful. I did not know about it. Could you try to use it @will-moore?